Bootstraps/Symfony.php

```diff
@@ -49,16 +49,24 @@ public function getApplication()
         $appAutoLoader = './app/autoload.php';
         if (file_exists($appAutoLoader)) {
             require $appAutoLoader;
-        } else {
+        } elseif (file_exists('./vendor/autoload.php')) {
             require './vendor/autoload.php';
         }
+        if(class_exists('\AppKernel')) {
+            $app = new \AppKernel($this->appenv, $this->debug);
+        } else {
+            if(file_exists('./app/bootstrap.php.cache') && file_exists('./app/AppKernel.php')) {
```

Comment:
The whole thing at line 55 should be replaced with

```php
if(file_exists('./app/bootstrap.php.cache')) {
    require_once './app/bootstrap.php.cache';
}
$app = new \AppKernel($this->appenv, $this->debug);
```

because if \AppKernel isn't there at this point requiring `app/AppKernel.php` would change that much as it will fail without an correctly configured autoloader anyway